src/components/v2v3/V2V3Project/V2V3Project.tsx

```diff
@@ -1,74 +1,38 @@
-import { Trans } from '@lingui/macro'
-import { Col, Row, Space } from 'antd'
+import { Row, Space } from 'antd'
 import ScrollToTopButton from 'components/buttons/ScrollToTopButton'
-import { TextButton } from 'components/buttons/TextButton'
 import { useModalFromUrlQuery } from 'components/modals/hooks/ModalFromUrlQuery'
-import { PayProjectForm } from 'components/Project/PayProjectForm'
 import { ProjectHeader } from 'components/Project/ProjectHeader'
 import { V2V3PayProjectFormProvider } from 'components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3PayProjectFormProvider'
-import VolumeChart from 'components/VolumeChart'
 import { ProjectMetadataContext } from 'contexts/shared/ProjectMetadataContext'
 import { V2V3ProjectContext } from 'contexts/v2v3/Project/V2V3ProjectContext'
 import { useIsUserAddress } from 'hooks/IsUserAddress'
 import { useHasNftRewards } from 'hooks/JB721Delegate/HasNftRewards'
 import useMobile from 'hooks/Mobile'
-import { useValidatePrimaryEthTerminal } from 'hooks/v2v3/ValidatePrimaryEthTerminal'
-import { useWallet } from 'hooks/Wallet'
-import { useContext, useState } from 'react'
-import { classNames } from 'utils/classNames'
-import { NftRewardsSection } from '../../NftRewards/NftRewardsSection'
+import { useContext } from 'react'
 import { ProjectBanners } from './banners/ProjectBanners'
-import { ManageNftsSection } from './ManageNftsSection/ManageNftsSection'
 import NewDeployModal, { NEW_DEPLOY_QUERY_PARAM } from './modals/NewDeployModal'
-import { V2V3ProjectTokenBalancesModal } from './modals/V2V3ProjectTokenBalancesModal'
-import ProjectActivity from './ProjectActivity'
-import TreasuryStats from './TreasuryStats'
-import { V2V3FundingCycleSection } from './V2V3FundingCycleSection'
-import { V2V3ManageTokensSection } from './V2V3ManageTokensSection'
+import { ProjectPageMobile } from './ProjectPageMobile'
+import { ProjectPageRightCol } from './ProjectPageRightCol'
+import { ProjectPageTabs } from './ProjectPageTabs/ProjectPageTabs'
```

Comment:
**note (non-blocking)** tech debt, should have a `index.ts` in there that exports this component